### PR TITLE
[color-gamut] Add reference tables to docs

### PR DESCRIPTION
**Pull request by DmitrySharabin**
- attributes & properties
- events
- CSS variables

### CODE REVIEW

File: src/color-gamut/README.md [hunk 1]
```diff
@@ -17,6 +17,33 @@ Dynamic:
...
+
+| Name | Description |
+|------|-------------|
+| `gamutchange` | Fired when the gamut changes for any reason, and once during initialization. |
+
+### CSS variables
+
+| Variable | Type | Description |
+|----------|---------------|-------------|
+| `--gamut-color` | `<color>` | Background color of gamut indicator. |
```
Comment (LeaVerou) [inline src/color-gamut/README.md:39]:
```suggestion
| `--gamut-color` | `<color>` | Background color of gamut indicator. Will override the color that depends on the actual gamut, so you should rarely use this directly. |
```

And move it to the end.

**Review by LeaVerou — CHANGES_REQUESTED** [review]
See comments

File: src/color-gamut/README.md [hunk 2]
```diff
@@ -17,6 +17,33 @@ Dynamic:
...
+| `--color-neutral-60` | `<color>` | Neutral background color of gamut indicator. |
```
[the remark this answers is not in the document]
Comment (DmitrySharabin) [reply src/color-gamut/README.md:40]: Done

File: src/color-gamut/README.md [hunk 3]
```diff
@@ -17,6 +17,33 @@ Dynamic:
...
+| `--color-green` | `<color>` | Green background color of gamut indicator. |
+| `--color-yellow` | `<color>` | Yellow background color of gamut indicator. |
+| `--color-orange` | `<color>` | Orange background color of gamut indicator. |
+| `--color-red` | `<color>` | Red background color of gamut indicator. |
+| `--color-red-dark` | `<color>` | Dark red background color of gamut indicator. |
```
[the remark this answers is not in the document]
Comment (DmitrySharabin) [reply src/color-gamut/README.md:45]: Done

File: src/color-gamut/README.md [hunk 4]
```diff
@@ -17,6 +17,33 @@ Dynamic:
 colorGamutElement.color = colorValue;
 ```
 
+## Reference
+
+### Attributes & Properties
+
+| Attribute | Property | Property type | Default value | Description |
+|-----------|----------|---------------|---------------|-------------|
+| `gamuts` | `gamuts` | `string` &#124; `Array<string>` &#124; `object` | `["srgb", "p3", "rec2020", "prophoto"]` | A list of gamuts to use. |
```
[the remark this answers is not in the document]
Comment (DmitrySharabin) [reply src/color-gamut/README.md:26]: Agreed that we don't have enough info here.

**Review by LeaVerou — APPROVED** [review]

File: src/color-gamut/README.md [hunk 5]
```diff
@@ -36,13 +36,12 @@ colorGamutElement.color = colorValue;
 
 | Variable | Type | Description |
 |----------|---------------|-------------|
-| `--gamut-color` | `<color>` | Background color of gamut indicator. Will override the color that depends on the actual gamut, so you should rarely use this directly. |
-| `--color-neutral-60` | `<color>` | Neutral background color of gamut indicator. |
```
Comment (LeaVerou) [inline src/color-gamut/README.md]: Gah, I didn’t mean remove this, I meant there needs to be a better variable here. I’m on it, keep an eye out so you can review.

**Pull request merged**